packages/telemetry/src/telemetry.ts

```diff
@@ -4,12 +4,15 @@ import * as os from 'os';
 import { sync as readPackageUpSync } from 'read-pkg-up';
 import { TelemetryClient, setup as AppInsights } from 'applicationinsights';
 import Conf from 'conf';
-import { ChildProcess, exec as execCallback, spawn as spawnCallback } from 'child_process';
+import { exec as execCallback } from 'child_process';
 import { promisify } from 'util';
 import { PathLike } from 'fs';
+import { CommandReturn } from '@appland/appmap/src/cmds/agentInstaller/commandStruct';
+import CommandStruct from '@appland/appmap/src/cmds/agentInstaller/commandStruct';
+import { run } from '@appland/appmap/src/cmds/agentInstaller/commandRunner';
+import { ChildProcessError } from '@appland/appmap/src/cmds/errors';
```

Comment:
This would mean anything importing telemetry also must include `@appland/appmap` as a dependency. It won't be handled automatically by dependency management tools because `@appland/telemetry` isn't a real package - it's imported via a symlink. 

Won't this break the `@appland/scanner` binary once published? I don't see any addition of `@appland/appmap` to the dependencies.

Reply:
It seems it will break it.  I think changing the symlink should be changed to be right under `src/` so that the full package name isn't needed.  Now addressed in https://github.com/getappmap/appmap-js/pull/1026/commits/2dc2649bf31ca31d8de922cbc7f52a65c6fbee1e